docs/_docs/features/hg.md

```diff
@@ -19,19 +19,18 @@ integration with support for [Mercurial](https://www.mercurial-scm.org/).
 
 ## Diff View
 
-The *Diff View* allows you to quickly see what has changed in the files you have modified. It
+The Diff View allows you to quickly see what has changed in the files you have modified. It
 shows what has changed from the current committed revision.
 
-If you `right-click` in the editor, you will see `Open in Diff View` (alternatively,
-`alt-cmd-shift-D` on Mac, `alt-ctrl-shift-D` on Linux).
+If you *right-click* in the editor, you will see the **Open in Diff View** option in the context-aware menu (alternatively, you can use `Alt-Cmd-Shift-D` on Mac or `Alt-Ctrl-Shift-D` on Linux).
 
 ![](/static/images/docs/feature-hg-diff-view-access.png)
 
-The Diff View is opened up in a new tab in the editor area. In the bottom pane are the files that
-have been modified, along with the Mercurial commit on which this diff is based. Click on the file
+The Diff View opens in a new tab in the [Editing Area](/docs/editor/basics/#editing). In the bottom pane are the files that
+have been modified, along with the Mercurial commit on which this diff is based. Click on a file
 in the bottom pane to see the changes for that file.
 
-In the left pane is what is currently committed for that file. In the right pane is what you have
+The left pane shows is what is currently committed for that file while the right pane displays what you have
```

Comment:
remove the "is" after "shows"

comma after "file", I think .... "file, while..."